src/main/java/seedu/edulink/logic/commands/EditCommand.java

```diff
@@ -285,6 +279,10 @@ public Optional<Set<Tag>> getTags() {
             return (tags != null) ? Optional.of(Collections.unmodifiableSet(tags)) : Optional.empty();
         }
 
+        public Optional<Set<Grade>> getGrades() {
```

Comment:
There may not be any need of the Optional. Can just return getGrades() of the student to be edited.